db-schema.sql

```diff
@@ -0,0 +1,47 @@
+create table plan(
+  "id" uuid UNIQUE DEFAULT uuid_generate_v4(),
+  "plan_id" character varying(100) NOT NULL,
+  "storage" bigint,
+  "created_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc'),
+  "updated_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc')
+);
+
+CREATE TRIGGER "plan_set_timestamp"
+BEFORE UPDATE ON "plan"
+FOR EACH ROW
+EXECUTE PROCEDURE trigger_set_timestamp();
+
+
+create table member_plan(
+  "id" uuid UNIQUE DEFAULT uuid_generate_v4(),
+  "creator" uuid REFERENCES "member" ("id") ON DELETE CASCADE,
+  "plan_id" uuid REFERENCES "plan" ("id"),
+  "customer_id" character varying(100),
+  "subscription_id" character varying(100),
+  "created_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc'),
+  "updated_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc')
+);
+
```

Comment:
I think we also wanted a unique member_id to avoid avoid multiple plan per user right?